.gitignore

```diff
@@ -27,6 +27,7 @@ build/
 .gradletasknamecache
 gradle-app.setting
 !gradle-wrapper.jar
+gradle/
```

Comment:
I think we shouldn't ignore this folder since it contains gradle-wrapper properties.